sekoiaio/info.json

```
@@ -17,15 +20,18 @@
                 "required": true,
                 "editable": true,
                 "visible": true,
-                "type": "text"
+                "type": "password",
```

This doesn't work, only type text works
```suggestion
                "type": "text",
```